io500.sh

```diff
@@ -2,43 +2,52 @@
 #
 # INSTRUCTIONS:
 # This script takes its parameters from the same .ini file as io500 binary.
+#
+# The only parts of the script that may need to be modified are:
+#  - setup_paths() to configure the binary locations and MPI parameters
+#  - setup_directories() to create/tune the IOR/mdtest output directories
+#
+# Please visit https://vi4io.org/io500-info-creator/ to help generate the
+# "system-information.txt" file, by pasting the output of the info-creator.
+# This file contains details of your system hardware for your submission.
 
+# Set the paths to the binaries and how to launch MPI jobs.
+# If you ran ./prepare.sh successfully, then binaries are in ./bin/
 function setup_paths {
-  # Set the paths to the binaries and how to launch MPI jobs.
-  # If you ran ./utilities/prepare.sh successfully, then binaries are in ./bin/
   io500_ior_cmd=$PWD/bin/ior
```

Comment:
We do not need this anymore as it is linked into the app directly.

Reply:
I guess the same is true for mdtest also, which is the whole point of the -app version...